frontend/www/js/omegaup/arena/course.ts

```diff
@@ -105,6 +115,41 @@ OmegaUp.on('ready', () => {
                 });
               });
           },
+          'new-clarification': ({
+            clarification,
+            clearForm,
+          }: {
+            clarification: types.Clarification;
+            clearForm: () => void;
+          }) => {
+            if (!clarification) {
+              return;
+            }
+            api.Clarification.create({
+              course_alias: payload.courseDetails.alias,
+              assignment_alias: payload.currentAssignment.alias,
+              problem_alias: clarification.problem_alias,
+              username: clarification.author,
+              message: clarification.message,
+            })
+              .then(() => {
+                clearForm();
+                refreshCourseClarifications({
+                  courseAlias: payload.courseDetails.alias,
+                } as CourseClarificationRequest);
```

Comment:
```suggestion
                });
```

aquí no hay necesidad de agregar la aseveración de tipo porque la función que se está mandando llamar tiene anotaciones de tipos, y se va a quejar si se le pasa algo que no debería...

... lo cual ocurre en este caso ^^;; así que en vez de el `as`, hay que ponerle el `type: CourseClarificationType.AllProblems` para que el checador de tipos esté contento. igual abajo.